src/contexts/store/index.ts

```diff
@@ -1,10 +1,12 @@
 import { configureStore } from "@reduxjs/toolkit";
 
 import { themeModeSlice } from "./slices/themeModeSlice";
+import { userSlice } from "./slices/userSlice";
 
 const store = configureStore({
   reducer: {
```

Comment:
root reducer powinien być wydzielony do oddzielnego pliku z użyciem combineReducers() - możemy to przegadać na callu